### ISSUE TITLE
SessionResult parameter is missing in the getResultOfPaymentSession method

### ISSUE THREAD

**Issue by GuiAmPm** [Adyen/adyen-node-api-library#1292]
Hello,

This method seems to be wrong:

https://github.com/Adyen/adyen-node-api-library/blob/283a26bd0c64da27ecd24c1a9f698acba582b903/src/services/checkout/paymentsApi.ts#L47

Calling it with only a SessionId returns an error because SessionResult is missing.
While the `SessionResult` parameter is mentioned in the JSDocs, it's missing in the method signature and is not being passed to the server

**Comment by RichardOestigaard** [comment]
Can confirm this issue on my side as well.

**Comment by wboereboom** [comment]
Hey @GuiAmPm , @RichardOestigaard ,

Thanks for raising this issue. We indeed also noticed the required query parameters, and we acknowledge it's been a hassle to include them through requestOptions.
We've added support for query parameters in method signatures wherever they've been included in our API specification. in #1304.
This will be included in our upcoming major release, as this will be a breaking change.

Kind Regards,
Wouter
Adyen
